File: CODES/1) C++/6) FOR/5.cpp
#include <iostream>

using namespace std;

int main()
{
	float k, x;
	cout << " KONFET NARXI = ";
	cin >> k;
	for (float i = 0.1; i < 1.1; i += 0.1)
	{
		x = k * i;
		cout << i << " KG NARXI = " << x << endl;
	}
	return 0;
}

File: LANGUAGES/4) Php/6) FOR/5.php
<?php
$n = 5000;
echo " bir kg konfet narxi = " . $n . " so'm <br>";
for ($i = 0.1; $i <= 1; $i += 0.1) {
    echo $i . " kg konfet narxi = " . $n * $i . "<br>";
}
